src/App.test.js

```diff
@@ -1,8 +1,16 @@
-import { render, screen } from "@testing-library/react";
+import React from "react";
+import { Provider } from "react-redux";
+import { render /*screen*/ } from "@testing-library/react";
+import { store } from "./app/store";
 import App from "./App";
 
 test("renders learn react link", () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+  render(
+    ""
+    // <Provider store={store}>
```

Comment:
should we remove these commented out lines from this test file?

Reply:
Done.  We can get them from the git commit history when we need to add them back, or I am sure a post on stack overflow would show how to add them.